tests/cnf/core/network/metallb/tests/common.go

```diff
@@ -284,3 +285,28 @@ func metalLbDaemonSetShouldMatchConditionAndBeInReadyState(
 	}, tsparams.DefaultTimeout, tsparams.DefaultRetryInterval).Should(expectedCondition, errorMessage)
 	Expect(metalLbDs.IsReady(120*time.Second)).To(BeTrue(), "MetalLb daemonSet is not Ready")
 }
+
+func createBGPPeerWithConnectTimeAndVerifyIfItsReady(connectTime metav1.Duration,
+	peerIP, bfdProfileName string, remoteAsn uint32, eBgpMultiHop bool, frrk8sPods []*pod.Builder) {
+	By("Creating BGP Peer")
+
+	bgpPeer := metallb.NewBPGPeerBuilder(APIClient, "testpeer", NetConfig.MlbOperatorNamespace,
+		peerIP, tsparams.LocalBGPASN, remoteAsn).WithPassword(tsparams.BGPPassword).WithEBGPMultiHop(eBgpMultiHop)
+
+	bgpPeer.WithConnectTime(connectTime)
```

Comment:
pls remove it and add  .WithConnectTime(connectTime) to the chain in the previos line

Reply:
changed